lib/db2.js

```diff
@@ -52,15 +52,27 @@ util.inherits(DB2, IBMDB);
  * @param {Function} [callback] The callback function
  */
 DB2.prototype.create = function(model, data, options, callback) {
+  debug('DB2.prototype.create: model=%j, where=%j, options=%j',
+        model, options);
```

Comment:
i think there is a `where` between `model` and `option`?
`debug('DB2.prototype.create: model=%j, where=%j, options=%j', model, where, options);`